pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -111,6 +111,18 @@ var (
 				owner: "125523088429",
 			},
 		},
+		providerconfigtypes.OperatingSystemRockyLinux: {
+			awstypes.CPUArchitectureX86_64: {
+				description: "Rocky-8-ec2-8*.x86_64",
+				// The AWS marketplace ID from Rocky Linux Community Platform Engineering (CPE)
+				owner: "792107900819",
+			},
+			awstypes.CPUArchitectureARM64: {
+				description: "Rocky-8-ec2-8*.x86_64",
```

Comment:
The architecture here is wrong one, should be 
```suggestion
				description: "Rocky-8-ec2-8*.aarch64",
```